types.go

```diff
@@ -38,6 +39,45 @@ var (
 	TimeType   = DType(pb.DType_TIME)
 )
 
+type SaveMode int
+
+func (mode SaveMode) GetNginxModeName() string {
+	switch mode {
+	case ErrorIfExists:
+		return ""
+	case Append:
+		return "CreateNewAttributes"
+	case Overwrite:
+		return ""
+	case Replace:
+		return "CreateOrReplaceAttributes"
```

Comment:
In case of replace, we need to write the nil values so that the attributes get deleted if already present in v3io.